dll/win32/browseui/internettoolbar.cpp

```diff
@@ -1504,8 +1504,14 @@ HRESULT STDMETHODCALLTYPE CInternetToolbar::AddBand(IUnknown *punk)
 
 HRESULT STDMETHODCALLTYPE CInternetToolbar::EnumBands(UINT uBand, DWORD *pdwBandID)
 {
-    UNIMPLEMENTED;
-    return E_NOTIMPL;
+    if (uBand == ~0ul)
```

Comment:
wouldn't using `-1` be the same? (and would the compiler emit a warning in this case?)

Reply:
Yes, it will warn

Reply:
yeah, MSVC is clever enough to understand ^^ (I've mostly seen the ~0 thing in wine code, this explains why :rofl:)